include/kframework/evm.md

```diff
@@ -2286,390 +2190,7 @@ There are several helpers for calculating gas (most of them also specified in th
            <nonce>   NONCE </nonce>
            <code>    CODE  </code>
            ...
-         </account>
-
-    syntax Bool ::= #accountEmpty ( AccountCode , Int , Int ) [function, total, klabel(accountEmpty), symbol]
```

Comment:
Perhaps these functions that reference `AccountCode` could stay in `evm.md`, and then `AccountCode` could stay in `evm.md`. I'm not sure if that is preferable.